### PR TITLE
add more unique key mangle role

### PR DESCRIPTION
**Pull request by kittizz**
## Proposed change
add more unique key mangle role

## Type of change

- [x] Bugfix
- [ ] New feature
- [ ] Code quality improvements to existing code or addition of tests
- [ ] Documentation

## Checklist

- [x] The code change is tested and works locally.
- [x] The code has been formatted using Black.
- [x] Tests have been added to verify that the new code works.
- [ ] Documentation added/updated if required.

### CODE REVIEW

File: custom_components/mikrotik_router/mikrotik_controller.py [hunk 1]
```diff
@@ -952,6 +954,12 @@ def get_mangle(self):
                     {"key": "dst-address"},
                     {"text": ":"},
                     {"key": "dst-port"},
+                    {"text": ","},
+                    {"key": "src-address-list"},
+                    {"text": ":"},
```
Comment (tomaae) [inline custom_components/mikrotik_router/mikrotik_controller.py:959]: should be "-" designating from-to
":" would be port

File: custom_components/mikrotik_router/switch.py [hunk 2]
```diff
@@ -596,7 +597,8 @@ async def async_turn_off(self) -> None:
             if self._ctrl.data["mangle"][uid]["uniq-id"] == (
                 f"{self._data['chain']},{self._data['action']},{self._data['protocol']},"
                 f"{self._data['src-address']}:{self._data['src-port']}-"
-                f"{self._data['dst-address']}:{self._data['dst-port']}"
+                f"{self._data['dst-address']}:{self._data['dst-port']},"
+                f"{self._data['src-address-list']}:{self._data['dst-address-list']}"
```
Comment (tomaae) [inline custom_components/mikrotik_router/switch.py:601]: should be "-" designating from-to
":" would be port

File: custom_components/mikrotik_router/switch.py [hunk 3]
```diff
@@ -578,7 +578,8 @@ async def async_turn_on(self) -> None:
...
```
Comment (tomaae) [inline custom_components/mikrotik_router/switch.py:582]: should be "-" designating from-to
":" would be port

**Comment by tomaae** [comment]
Looks good, just a small change is needed

**Comment by kittizz** [comment]
ok

**Comment by tomaae** [comment]
Thanks.
I will wait a little bit before merging, since I want to go over additional possible changes for this UID. Dont want to change UID twice for people who use master branch.

**Review by tomaae — APPROVED** [review]

**Pull request merged**